src/gw2wiki/claimTicketOffers.js

```diff
@@ -1,22 +1,12 @@
-import { getWikiHtml } from './helpers.js'
-import execAll from 'execall'
+import { getSmwQueryResult } from './helpers.js'
 
 // Get the current claim ticket offers
 export default async function claimTicketOffers () {
-  let page = await getWikiHtml('Template:Inventory/black_lion_claim_ticket')
+  const result = await getSmwQueryResult('[[Has vendor::Black Lion Claim Ticket]][[Has item cost.Has item currency::Black Lion Claim Ticket]]|?Sells item=item|?Has item cost.Has item value=value')
```

Comment:
- Why does this use a different type of query ("has item value" vs "has item cost")?
- How come this does not match [these weapons](https://wiki.guildwars2.com/wiki/Black_Lion_Weapons_Specialist/historical#Abaddon's_Weapons)? (It's correct it's not matching, I'm just wondering why)

Reply:
1. `Has item cost` is a composite property, containing `Has item value` and `Has item currency`. We are filtering all entries to have `Has item cost.Has item currency::Black Lion Claim Ticket`. Since we know that `Has item cost.Has item currency` will always be the same, we only need `Has item cost.Has item value`. For some reason the mediawiki API did not always return  `Has item cost.Has item value` for the statuettes, even though it is set. Instead I now query the composite property `Has item value` and then accessing the sub property.  
   I'm not sure why it doesn't work for the statuettes, but does for claim tickets. It should always work.
   I've added tests for boths that catches this issue, so even if the other query should break one day, the test will catch that.
2. The first part of the query (`[[Has vendor::Black Lion Claim Ticket]]`) only selects entries that have the vendor "Black Lion Claim Ticket". The wiki sets the vendor to the page title, so we don't get historical ones, because those entries instead have `[[Has vendor::Black Lion Weapons Specialist/historical]]`